### DISCUSSION TITLE
[Bug]: Child inherited field are null

### DISCUSSION THREAD

**Discussion by axessandro** [pimcore/pimcore#16079] [Q&A / Support] [answered]
### Pimcore version

11.0.5

### Steps to reproduce

Create data object (in this example “Style”), and then create a child that inheritance “styleCode” field.

Check allow inheritance setting in class Style.

Create a Bundle using PimcoreBundleGeneratorBundle.

In Default controller of bundle already created: get the child using his id and then take the inherited field. 

**Attention**
I test this snippet in other controllers and command this happen only in custom bundle

```
$child = Style::getById(1848);
dd($child->getStyleCode());
```

In Pimcore interface the field appear inherited

<img width="372" alt="Screenshot 2023-10-10 alle 10 23 19" src="https://github.com/pimcore/pimcore/assets/104524529/a66a3e8e-2acc-4ed3-9d99-38b73a4c3946">

### Actual Behavior

dd response will be null

### Expected Behavior

dd response take the inherited field from the parent

**Comment by dvesh3** [comment, marked as answer]
please use `DataObject::setGetInheritedValues(true);` as explained [here](https://pimcore.com/docs/pimcore/10.6/Development_Documentation/Objects/Object_Classes/Data_Types/Fieldcollections.html#page_Inheritance).
